README.md

```diff
@@ -6,14 +6,14 @@
 
 # Ankaios Python SDK for Eclipse Ankaios
```

Comment:
```suggestion
# Python SDK for Eclipse Ankaios
```

Having Ankaios two times does not sound that good.

Reply:
Done.